src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -6,6 +6,11 @@ Loader {
     id: statusSmartIdenticon
 
     property string name: ""
+    property bool isBadgeVisible: false
+
+    // Badge color properties must be set if isBadgeVisible = true
+    property color badgeColor
```

Comment:
Here we could expose the complete `badge` component to have more flexibility. What do you think is it a better option @PascalPrecht @Khushboo-dev-cpp ?

Reply:
Yes, I think that's a good idea. Rather introduce the entire badge so we don't need to reintroduce all the additional properties just to propagate them.